src/sdk/index.ts

```diff
@@ -0,0 +1,4 @@
+import SDK from './SDK';
+import { useSDK } from './useSDK';
+
+export { SDK, useSDK };
```

Comment:
```
export SDK from './SDK';
export { useSDK } from './useSDK';
```

does this work? sorry i'm being annoying lol

Reply:
done here: https://github.com/MetaMask/test-dapp-multichain/pull/20/commits/c5102b213e10b729fcc2fb63164f1af1cb4f9277